crypto/src/e2e_identity/identity.rs

```diff
@@ -259,6 +259,114 @@ mod tests {
         .await
     }
 
+    // this test is a duplicate of its counterpart but taking federation into account
+    // The heavy lifting of cross-signing the certificates is being done by the test utils.
+    #[async_std::test]
+    #[wasm_bindgen_test]
+    async fn should_read_revoked_device_cross_signed() {
```

Comment:
This test is exactly the same as `should_read_revoked_device`, the only thing different looks to be the name. It feels like something is missing.